pages/system/tables.js

```diff
@@ -87,27 +87,25 @@ import { TableContent, TableColumn } from 'slate-react-system';`}
         <System.H2>Output</System.H2>
         <hr />
         <br />
-        <Group title="Table example">
-          <System.Table
-            data={{
-              columns: [
-                { key: 'a', name: 'Link', type: 'FILE_LINK' },
-                { key: 'b', name: 'Value', width: '88px' },
-                { key: 'c', name: 'Tooltip', tooltip: 'A tooltip.', width: '128px' },
-                { key: 'd', name: 'Copyable', copyable: true, width: '88px' },
-              ],
-              rows: [
-                { id: 1, a: 'col 1 row 1', b: 'col 1 row 2', c: 'col 1 row 3', d: 'col 1 row 4' },
-                { id: 2, a: 'col 2 row 1', b: 'col 2 row 2', c: 'col 2 row 3', d: 'col 2 row 4' },
-                { id: 3, a: 'col 3 row 1', b: 'col 3 row 2', c: 'col 3 row 3', d: 'col 3 row 4' },
-                { id: 3, a: 'col 4 row 1', b: 'col 4 row 2', c: 'col 4 row 3', d: 'col 4 row 4' },
-              ],
-            }}
-            selectedRowId={this.state.table_data}
-            onChange={this._handleChange}
-            name="table_data"
-          />
-        </Group>
+        <System.Table
+          data={{
+            columns: [
+              { key: 'a', name: 'Link', type: 'FILE_LINK' },
+              { key: 'b', name: 'Value', width: '88px' },
+              { key: 'c', name: 'Tooltip', tooltip: 'A tooltip.', width: '128px' },
+              { key: 'd', name: 'Copyable', copyable: true, width: '88px' },
+            ],
+            rows: [
+              { id: 1, a: 'col 1 row 1', b: 'col 1 row 2', c: 'col 1 row 3', d: 'col 1 row 4' },
+              { id: 2, a: 'col 2 row 1', b: 'col 2 row 2', c: 'col 2 row 3', d: 'col 2 row 4' },
+              { id: 3, a: 'col 3 row 1', b: 'col 3 row 2', c: 'col 3 row 3', d: 'col 3 row 4' },
+              { id: 3, a: 'col 4 row 1', b: 'col 4 row 2', c: 'col 4 row 3', d: 'col 4 row 4' },
+            ],
+          }}
+          selectedRowId={this.state.table_data}
```

Comment:
If you're going to have multiple tables on the same page, you're going to need multiple `name` and multiple `selectedRowId` state variables.

Let me know if this is confusing or doesn't make sense.